sync_first/sync_first/urls.py

```diff
@@ -15,7 +15,12 @@
 """
 from django.conf.urls import url
 from django.contrib import admin
+from sync_first_app import views
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^search', views.show_search_page),
```

Comment:
A bit of name improving needed.
For example serach_person, new_incident...
If we will need one day another search or anything new -  there will be a collision.